pkg/bot/slack.go

```diff
@@ -320,12 +348,37 @@ func (b *Slack) SendMessage(ctx context.Context, msg string) error {
 			errs = multierror.Append(errs, fmt.Errorf("while sending Slack message to channel %q: %w", channelName, err))
 			continue
 		}
-		b.log.Debugf("Message successfully sent to channel %b at %b", channelID, timestamp)
+		b.log.Debugf("Message successfully sent to channel %q at %q", channelID, timestamp)
+	}
+
+	return errs.ErrorOrNil()
+}
+
+// SendInteractiveMessage sends message with interactive sections to Slack channels.
+func (b *Slack) SendInteractiveMessage(ctx context.Context, msg interactive.Message) error {
+	errs := multierror.New()
+	for _, channel := range b.getChannels() {
+		channelName := channel.Name
+		b.log.Debugf(">> Sending message to channel %q: %+v", channelName, msg)
```

Comment:
nit: I think we can get rid of `>>` in logs 🤔

Reply:
It was there already, so for consistency with other logs I left them, but if we agree I will remove it in all places, OK?